release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1678,10 +1678,30 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, machines created by a control plane machine set were considered ready once the underlying Machine API machine was running. With this release, the machine is not considered ready until the node linked to that machine is also ready. (link:https://issues.redhat.com/browse/OCPBUGS-7989[*OCPBUGS-7989*])
+
+* Previously, the Control Plane Machine Set Operator prioritized failure domains alphabetically and moved machines from alphabetically later failure domains to alphabetically earlier failure domains, even if doing so did not improve the the availability of the machines across the failure domains. With this release, the Operator is updated to prioritize failure domains that are present in the existing machines and respect existing failure domains that provide better availability. (link:https://issues.redhat.com/browse/OCPBUGS-7921[*OCPBUGS-7921*])
+
+* Previously, when a control plane machine on a vSphere cluster that uses a control plane machine set was deleted, sometimes two replacement machines were created. With this release, the control plane machine set no longer causes an extra machine to be created. (link:https://issues.redhat.com/browse/OCPBUGS-7516[*OCPBUGS-7516*])
+
+* Previously, when the availability zone and subnet ID in a machine set were mismatched, a machine was created successfully using the machine set specification with no indication to the user of the mismatch. Because the mismatched values can cause problems with some configurations, this occurrence should be visible as a warning message. With this release, a warning about the mismatch is logged. (link:https://issues.redhat.com/browse/OCPBUGS-6882[*OCPBUGS-6882*])
+
+* Previously, when creating an {product-title} cluster on Nutanix that uses DHCP instead of an IP address management (IPAM) network configuration, the hostname of the VM was not set by DHCP. With this release, the VM hostname is set with values from the ignition configuration files. As a result, the issue is resolved for DHCP as well as other network configuration types. (link:https://issues.redhat.com/browse/OCPBUGS-6727[*OCPBUGS-6727*])
+
+* Previously, multiple clusters could be created in the `openshift-cluster-api` namespace. This namespace should contain only one cluster. With this release, additional clusters cannot be created in this namespace. (link:https://issues.redhat.com/browse/OCPBUGS-4147[*OCPBUGS-4147*])
```

Comment:
```suggestion
* Previously, multiple clusters could be created in the `openshift-cluster-api` namespace. This namespace must contain only one cluster. With this release, additional clusters cannot be created in this namespace. (link:https://issues.redhat.com/browse/OCPBUGS-4147[*OCPBUGS-4147*])
```
Per IBM Style, avoid `should` and don't use it as a synonym for `must`. The word `should` can have an ambiguous meaning.